### PR TITLE
Initialize rails proyect

### PR DESCRIPTION
**Pull request by Zama15**
Description

What did you implemented/Why did you implemented this:

Initialize the project and setup the datase using the PostgreSQL
For more information see issue #2

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+gem 'sass-rails'
+
+# Use Redis adapter to run Action Cable in production
+# gem "redis", ">= 4.0.1"
+
+# Use Kredis to get higher-level data types in Redis [https://github.com/rails/kredis]
+# gem "kredis"
+
+# Use Active Model has_secure_password [https://guides.rubyonrails.org/active_model_basics.html#securepassword]
+# gem "bcrypt", "~> 3.1.7"
```
Comment (juperameza) [inline Gemfile]: Dont leave comment you wont use

File: test/channels/application_cable/connection_test.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+module ApplicationCable
+  class ConnectionTest < ActionCable::Connection::TestCase
+    # test "connects with cookies" do
+    #   cookies.signed[:user_id] = 42
+    #
+    #   connect
+    #
+    #   assert_equal connection.user_id, "42"
+    # end
```
Comment (juperameza) [inline test/channels/application_cable/connection_test.rb]: Delete code you wont use

**Review by juperameza — CHANGES_REQUESTED** [review]

**Comment by MauricioNC** [comment]
Always remove the text of the template, it's ok if you takes this text as reference, but once you described all the changes for the PR, remove this template text

File: Gemfile [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+# Reduces boot times through caching; required in config/boot.rb
+gem "bootsnap", require: false
+
+# Use Active Storage variants [https://guides.rubyonrails.org/active_storage_overview.html#transforming-images]
+# gem "image_processing", "~> 1.2"
+
+group :development, :test do
+  # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
+  gem "debug", platforms: %i[ mri mswin mswin64 mingw x64_mingw ]
+end
```
Comment (JuanequeX) [inline Gemfile]: remove this gem an use `pry-rails` to debugging

File: config/credentials.yml.enc [hunk 4]
```diff
@@ -0,0 +1 @@
+c6CMp8hY2684ybgcOkT3QL/VG8fJtLCBvnyNdHSTGKldEnK2uRoCCkrbIWSG1kPbc97nY3qptcXSi3m00jtDvoq9bIjWaRAPWbhWQYWoUrjrFQcb9HfD9xYbViGoXWzre+82Qi1kQYleUAfHu7gn8V6Cx8q44FTDUCy9BexDIoa6boYNtNFFVPo5WSwrDOzxp2ERG3isGi9j+YgCqE0OvrWQlb9UqvW1sFW/r+EZoW2HEhDh69fSBwTGxCkGDYVvCdOaWsw1C5XPdA1hzZcg6LgzgFaD0vX1Iy1yjnqlqEMwj6iKPNyj/bmmEvPEAEPG4/9Lncgx64CmrzhCEGw3uDYtGEZdCo3bdQu/XAjte3MeiK2AtQXgiEGFzGtwHX9Odw2INqIJg+NN32mqJqnRqGyMft9V--YytIosuY99vbHs60--ZIIZTdVpuj40lblhI+h8ng==
```
Comment (JuanequeX) [inline config/credentials.yml.enc]: Add this credential to your `.gitignore`

**Review by JuanequeX — CHANGES_REQUESTED** [review]

**Comment by JuanequeX** [comment]
Use a correct format to describe what you do on your pull requests, pushing in this branch this [template](https://gist.github.com/JuanequeX/1cfdf24e2a731bb9b50b0c7cf7a14753)

File: config/credentials.yml.enc (hunk 4 above)
Comment (Zama15) [reply config/credentials.yml.enc]: and what it does the line?

File: test/channels/application_cable/connection_test.rb (hunk 2 above)
Comment (Zama15) [reply test/channels/application_cable/connection_test.rb]: done

File: Gemfile (hunk 3 above)
Comment (Zama15) [reply Gemfile]: okey

File: .gitignore [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+!/tmp/storage/
+!/tmp/storage/.keep
+
+/public/assets
+
+# Ignore master key for decrypting credentials and more.
+/config/master.key
+
+# Ignore credentials
+c6CMp8hY2684ybgcOkT3QL/VG8fJtLCBvnyNdHSTGKldEnK2uRoCCkrbIWSG1kPbc97nY3qptcXSi3m00jtDvoq9bIjWaRAPWbhWQYWoUrjrFQcb9HfD9xYbViGoXWzre+82Qi1kQYleUAfHu7gn8V6Cx8q44FTDUCy9BexDIoa6boYNtNFFVPo5WSwrDOzxp2ERG3isGi9j+YgCqE0OvrWQlb9UqvW1sFW/r+EZoW2HEhDh69fSBwTGxCkGDYVvCdOaWsw1C5XPdA1hzZcg6LgzgFaD0vX1Iy1yjnqlqEMwj6iKPNyj/bmmEvPEAEPG4/9Lncgx64CmrzhCEGw3uDYtGEZdCo3bdQu/XAjte3MeiK2AtQXgiEGFzGtwHX9Odw2INqIJg+NN32mqJqnRqGyMft9V--YytIosuY99vbHs60--ZIIZTdVpuj40lblhI+h8ng==
```
Comment (juperameza) [inline .gitignore]: Thats not how you do it and also add a new line at the end of the file

**Review by juperameza — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 5 above)
Comment (Zama15) [reply .gitignore]: done

**Review by juperameza — APPROVED** [review]
LGTM

**Pull request merged**